prober/grpc_health_v1.go

```diff
@@ -0,0 +1,50 @@
+package prober
+
+import (
+	"context"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/prometheus/blackbox_exporter/config"
+
+	"google.golang.org/grpc"
+	healthpb "google.golang.org/grpc/health/grpc_health_v1"
+)
+
+func ProbeGRPCHEALTHV1(ctx context.Context, target string, module config.Module, registry *prometheus.Registry, logger log.Logger) bool {
+	var (
+		conn    *grpc.ClientConn
+		parts   []string
+		service string
+		host    string
+	)
+
+	parts          = strings.Split(target, "/")
+	host, service  = parts[0], parts[1]
```

Comment:
I don't think `service` should be part of the target. It should be a field in the probe like `method` is in the `http` probe for example.

So this would end up as a field in the GRPCHealthV1Probe struct, although I'm not quite sure how that value gets into this function.

Reply:
Nice. Was looking for something like this. Only found other args passed via module definition. This is more flexible.